src/sys/fuchsia/ready.rs

```diff
@@ -0,0 +1,138 @@
+use event_imp::{Ready, ready_as_usize, ready_from_usize};
+pub use magenta_sys::mx_signals_t;
+use std::ops;
+
+// The following impls are valid because Fuchsia and mio both represent
+// "readable" as `1 << 0` and "writable" as `1 << 2`.
```

Comment:
Sure! I don't think it's going to change anytime soon, but better to be safe than wind up with untraceable errors.